feature-detects/dom/shadowroot.js

```diff
@@ -0,0 +1,22 @@
+/*!
+{
+  "name": "Shadow DOM API",
+  "property": ["shadowroot", "shadowrootlegacy"],
+  "notes": [
+    {
+      "name": "MDN Docs",
+      "href": "https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot"
+    }
+  ],
+  "authors": ["Kevin Coyle", "Pascal Lim"],
+  "tags": ["dom"]
+  }
```

Comment:
please add the caniuse references too https://caniuse.com/#feat=shadowdom https://caniuse.com/#feat=shadowdomv1